h/api/presenters.py

```diff
@@ -28,6 +38,28 @@ def links(self):
                 out[name] = link
         return out
 
+    @property
+    def text(self):
+        if self.annotation.text:
+            return self.annotation.text
+        else:
+            return ''
+
+    @property
+    def tags(self):
+        if self.annotation.tags:
+            return self.annotation.tags
+        else:
+            return []
+
+    @property
+    def target(self):
+        target = {'source': self.annotation.target_uri}
+        if self.annotation.target_selectors:
+            target['selector'] = self.annotation.target_selectors
+
+        return [target]
```

Comment:
The examples in https://www.w3.org/TR/annotation-model/ use a single object rather than an array for the case where an annotation has a single target. Should we do the same?